src/block.ts

```diff
@@ -1,15 +1,18 @@
-import { BlockHeader } from "./blockHeader";
+import {BlockHeader} from "./blockHeader";
```

Comment:
I changed this inconsistency in my PR yesterday. Please make sure to change the import to `import { BlockHeader } from`.